### DISCUSSION TITLE
Welcome to Utilities Discussions!

### DISCUSSION THREAD

**Discussion by legendhimself** [acm-mjct/Utilities#70] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Mohammed-bm** [comment]
hi

**Reply by muttu4252** [reply]
hi

**Reply by ayankhan03** [reply]
hello!!

**Reply by nazish-16** [reply]
Hola..👋

**Comment by abdul-rehmant** [comment]
html

**Comment by tallha-khan** [comment]
hi

**Comment by umayman** [comment]
hi

**Comment by IQRATABASSUM1234** [comment]
hello

**Comment by abdul-rehmant** [comment]
slow down

**Comment by ghost** [comment]
How does one compile the binaries into a useable code or program ?

**Comment by DRAGONXBLITZ** [comment]
Hi vroooooooo

**Comment by tallha-khan** [comment]
bruh

**Comment by nazish-16** [comment]
When is the acm/mjcet website being updated and relaunched?
